src/labels/CustomLabels.labels

```diff
@@ -2736,19 +2736,33 @@ To check your GAU Allocation settings, go to Donations | GAU Allocations.</value
         <shortDescription>Help text to indicate user is creating a new opportunity</shortDescription>
         <value>You are currently creating a new Opportunity.</value>
     </labels>
+    <labels>
+        <fullName>bgeFormNewPaymentHelptext</fullName>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Help text to indicate user is applying a new payment to an opportunity</shortDescription>
+        <value>You are currently applying a new Payment to Opportunity:</value>
```

Comment:
@smarovich is this label ok? The name of the opportunity appears after the semi-colon here.